README.md

```diff
@@ -84,7 +103,8 @@ Define a global variable instructor and set it equal to your Squad Instructor's
 
 Your Answer:
 ```js
-// write code here
+instructor = Becky; //global variable
```

Comment:
Just don't forget that you should be assigning it to a string in this case since we haven't defined a variable Becky, instead of `instructor = Becky`, it should be` instructor = "Becky"`